app/src/main/java/com/example/android/uamp/viewmodels/MainActivityViewModel.kt

```diff
@@ -164,12 +167,23 @@ class MainActivityViewModel(
         }
     }
 
+    /**
+     * Sends a command via MusicServiceConnection to MusicService to toggle spatial audio
+     *
+     * @param enable current toggle state of spatial audio
+     */
+    suspend fun toggleSpatialization(enable: Boolean){
+        val bundle = Bundle()
```

Comment:
FYI: The Kotlin convention for something like this where you create a new object and then set various properties on it is to use apply like:

```
val bundle = Bundle().apply {
    putBoolean(EXTRAS_TOGGLE_SPATIALIZATION, enable)
}
```

In cases like this where you're literally only setting one value, I think either setting it like you did or using apply is fine, so this comment is just an FYI about a Kotlin feature :)

Reply:
Awesome! This is much cleaner. :)